integrations/redis/clients.go

```diff
@@ -63,8 +64,23 @@ type Token struct {
 	KeyFunc func(string) string
 }
 
-func externalClient(ctx context.Context) (*redis.Client, error) {
-	addr := string(sdkmodule.FunctionDataFromContext(ctx))
+func (m *module) externalClient(ctx context.Context) (*redis.Client, error) {
+	cid, err := sdkmodule.FunctionConnectionIDFromContext(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	vars, err := m.vars.Get(ctx, sdktypes.NewVarScopeID(cid))
+	if err != nil {
+		return nil, err
+	}
+
+	addrVar := vars.Get(sdktypes.NewSymbol("addr"))
```

Comment:
Document this in docs/autokitteh.yaml

Reply:
done

Reply:
nit: please rename "addr" to "connection_string" (or something like that) - the URI contains more (optional) parameters beyond the address.

Reply:
i changed it to `dsn`. i see they call it `url`, will change to `url`.